### PR TITLE
Hides the login form and shows the spinner when doing GISF auto login

### PR DESCRIPTION
**Pull request by jaycee**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6701/

File: jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js [hunk 1]
```diff
@@ -41,6 +41,7 @@ YUI.add('usso-login-link', function() {
         e.preventDefault();
       }
       this.props.loginToController(err => {
+        console.log('In login');
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js]: Leftover.

File: jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js [hunk 2]
```diff
@@ -59,6 +60,7 @@ YUI.add('usso-login-link', function() {
         }
 
         const callback = this.props.callback;
+        console.log('CALLBACK' + callback);
```
Comment (frankban) [inline jujugui/static/gui/src/app/components/usso-login-link/usso-login-link.js]: Leftover.

**Comment by jaycee** [comment]
QA:

Spin up the gui either with the ini set to connect to jimm and gisf set to true, or with `guiproxy -config gisf:true` with the staging jimm controller.

Go to /login; you should see the spinner stay up while the gui opens the second tab for login. Once you've logged in the spinner should be gone and you should see the canvas. You should never see the login form.

Turn gisf off (either in INI or restart guiproxy w/o the gisf flag). Go to /login. The form should appear.

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6705/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6707/

**Comment by frankban** [comment]
QA ok.

**Comment by Lukewh** [comment]
QA OK

**Comment by jaycee** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**